cdap-tms/src/main/java/io/cdap/cdap/messaging/service/CoreMessageFetcher.java

```diff
@@ -33,26 +34,29 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Implementation of {@link MessageFetcher} that fetch messages directly from {@link MessageTable}
- * and {@link PayloadTable}.
+ * Fetch messages directly from {@link MessageTable} and {@link PayloadTable}.
  */
-final class CoreMessageFetcher extends MessageFetcher {
+final class CoreMessageFetcher {
```

Comment:
Please drop this class I believe it's not used anymore

Reply:
Oh right. Thanks for catching this.